hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/curator/ZKCuratorManager.java

```diff
@@ -135,29 +135,51 @@ public void start() throws IOException{
    * @param authInfos List of authentication keys.
    * @throws IOException If the connection cannot be started.
    */
-  public void start(List<AuthInfo> authInfos) throws IOException {
+  public void start(List<AuthInfo> authInfos) throws IOException{
```

Comment:
checkstyle issue 

```
public void start(List<AuthInfo> authInfos) throws IOException { 
...
}
```

Reply:
Thank you very much for your help, I will fix it.